sklearn/decomposition/pca.py

```diff
@@ -216,6 +216,68 @@ def __init__(self, n_components=None, copy=True, whiten=False):
         self.copy = copy
         self.whiten = whiten
 
+    def get_feature_names(self, input_features=None, full_output=False):
+        """
+        Return dominant feature names for each component
+
+        Parameters
+        ----------
+        input_features : list of string, shape (n_features), optional
+            String names for input features if available. By default,
+            "x0", "x1", ... "xn_features" is used.
+
+        full_output : boolean, default False
```

Comment:
I thought about this independently, and disagree. These are not parameters that need `set_params` etc. functionality, and can be mediated in a pipeline by `**kwargs` with name prefixing.